app/controllers/api/v1/items/finder_controller.rb

```diff
@@ -2,7 +2,7 @@ class Api::V1::Items::FinderController < ApplicationController
   before_action :format_unit_price, only: [:index, :show]
 
   def show
-    render json: Item.find_by(item_params)
+    render json: Item.where(item_params).first
```

Comment:
@meyerhoferc @scmountain I'm not positive what's happening here. It's an issue that a few people are running into, and I agree that it's not something that makes a lot of sense. I'd prefer that you keep the `find_by` for now, even if that results in a failing test for the spec harness.